Dockerfile

```
@@ -28,3 +28,4 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o ./machine-controller-ma
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 WORKDIR /
 COPY --from=builder /go/src/sigs.k8s.io/cluster-api-provider-azure/machine-controller-manager .
+COPY --from=builder /go/src/sigs.k8s.io/cluster-api-provider-azure/bin/termination-handler .
```

Bin dir doesn't exist
```suggestion
COPY --from=builder /go/src/sigs.k8s.io/cluster-api-provider-azure/termination-handler .
```

---

@alexander-demichev This is meant to copy but as far as I can see the binary is never built in this dockerfile.

---

We run `make build` before

---

We don't run `make build` in this dockerfile, but we should try to make it consistent with the others, so you could change it to do so

---

Ohh, sorry then. I didn't think it's not consistent with other providers.